modules/event_routing/event_routing.c

```diff
@@ -288,6 +288,16 @@ int fixup_wait(void** param, int param_no)
 static int notify_on_event(struct sip_msg *msg, void *ev, void *avp_filter,
 									void *route, void *timeout)
 {
+	int t = 0;
+
+	if (timeout) {
+		gparam_p gp;
```

Comment:
@fgast , in terms of coding style we do not encourage / favor variable declarations  in the beginning of blocks . Please move it to the beginning of the functions, here and in the wait_for_event(). Otherwise the PR looks good to me.

Reply:
@bogdan-iancu I updated the PR.